modules/mapbox/src/deck-utils.ts

```diff
@@ -33,7 +33,7 @@ export function getDeckInstance({
   const customRender = deck?.props._customRender;
 
   const deckProps: DeckProps = {
-    useDevicePixels: true,
+    useDevicePixels: (deck && deck.props.useDevicePixels) || true,
```

Comment:
This expression will always be true

Reply:
This also wouldn't won't work since `useDevicePixels` can be validly be set to `false`. Do you have a preferred way to check if the object is set (just _not_ undefined)?

Reply:
`'useDevicePixels' in deck.props` should do it.